password_security/models/res_company.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 LasLabs Inc.
+# Copyright 2017 Kaushal Prajapati <kbprajapati@live.com>.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    password_expiration = fields.Integer(
+        "Days",
+        default=60,
```

Comment:
```suggestion
```
I don't know what the solution is unfortunately. I tried setting all the defaults to zero and setting the company values to the "correct" defaults on each of the tests but then the main company will have default 0 values.

I thought maybe we could use ir.defaults to put the defaults back, but the default values will only be set on a new company creation.

We can't use a post_install_hook since test suites are run with a post-install tag...

Do you have any suggestions?

Reply:
I would go for setting `password_expiration` default to 0 and update the readme and the tests accordingly. What do you think?

Do you know if `password_expiration` the only field to be fixed?

Reply:
Unfortunately it's all of them 🥲
![base test errors](https://github.com/OCA/server-auth/assets/22336846/23b7569a-0e35-4026-a0ec-7e2472bf24cf)
![check password error](https://github.com/OCA/server-auth/assets/22336846/bc69ec48-a531-4e7a-bf71-bc836089ce60)

It's because the base test suite passwords don't fit the password requirements...

Reply:
See https://github.com/odoo/odoo/pull/114858 for some commentary from Odoo developers on this problem.

Reply:
Regarding this problem, for the time being, I would revert https://github.com/OCA/server-auth/pull/482/commits/58adcf51fd527c1621b77b2d60842e89dd6d7ad9 and I would avoid to solve the problem within this PR. Later on we could open a separate PR in case https://github.com/odoo/odoo/pull/114858 will not go any further.
Is that ok with you all?

Reply:
Makes sense. I think I'll end up having to make a custom module to prevent test failures on my end for now.

Reply:
@astirpe How do you feel about having an `enable_password_security` field on res.config.settings that is related to company_id?

Then we can enable it for tests and have the setting_box attr be invisible depending on it

Reply:
@hwangh95 seems a good idea to me, however I'm very busy on a project these days. Is there any possibility for you to take over this PR for continuing the development? Would be really appreciated and I could review it later on.